components/categories/categories.tsx

```diff
@@ -7,9 +9,11 @@ export default function Categories() {
   return (
     <div className={styles['categories']}>
       {categoriesArray.map(({ name, color, href }) => (
-        <Button key={name} buttonColor={color} href={href}>
-          {name}
-        </Button>
+        <Link href='' key={name}>
+          <Button buttonColor={color} href={href}>
```

Comment:
With `Next` you can add `passHref` to the Link, and it will pass that to whatever is below, so it should be:
```
<Link href={href} key={name} passHref>
      <Button buttonColor={color}>
```
and also making the `href` prop optional for the `Button` component